synapse/storage/room.py

```diff
@@ -533,73 +533,92 @@ def block_room(self, room_id, user_id):
         )
         self.is_room_blocked.invalidate((room_id,))
 
+    def get_media_mxcs_in_room(self, room_id):
+        def _get_media_ids_in_room(txn):
```

Comment:
it's confusing that this is called `_get_media_ids_in_room` when it gets mxcs rather than ids. 

The convention we use elsewhere is to add `_txn` to the end of the name of the wrapper function (see [store_room_txn](https://github.com/turt2live/synapse/blob/a94d9b6b825c6b2db375460268567e637e10709a/synapse/storage/room.py#L58) for example). It doesn't need a leading `_` if it's an inner function.